src/app/pages/send/send-crypto-asset-form/components/recipient-accounts-drawer/recipient-accounts-drawer.tsx

```diff
@@ -15,12 +18,15 @@ const smallNumberOfAccountsToRenderWholeList = 10;
 export const RecipientAccountsDrawer = memo(() => {
   const { whenWallet } = useWalletType();
   const accounts = useStacksAccounts();
+  const currentAccount = useCurrentStacksAccount();
   const navigate = useNavigate();
 
   const onGoBack = useCallback(() => navigate('..', { replace: true }), [navigate]);
 
   if (!accounts) return null;
 
+  accounts.splice(currentAccount?.index || 0, 1);
```

Comment:
Could you instead use a `filter` operation on the array to remove the current account?